### ISSUE TITLE
"Filter all spaces" placeholder is lighter than it should be

### ISSUE THREAD

**Issue by robintown** [element-hq/element-web#19470]
### Steps to reproduce

Look at the "Filter all spaces" input

### Outcome

![filter](https://user-images.githubusercontent.com/48614497/138346200-5882596a-fc48-4512-800a-78d8ebbd94f0.png)

#### What did you expect?

The "Filter all spaces" text should be of a heavier font weight

#### What happened instead?

The text is of a very light font weight, making it difficult to read. This is a recent regression.

### Operating system

NixOS unstable

### Browser information

Firefox 92.0.1

### URL for webapp

develop.element.io

### Homeserver

townsendandsmith.ml

### Will you send logs?

No
